Project/core/src/main/java/Controller/GameController.java

```diff
@@ -86,10 +92,37 @@ public GameController(Csv boardCsv, ArrayList<User> users) {
         initController(gameState);
     }
 
+    /**
+     * Returns wether the board should be active or not
+     * 
+     * @return boolean, true if active and false if not
+     */
+    public boolean getBoardIsActive() {
+        return this.boardIsActive;
+    }
+
+    /**
+     * Setter for boardIsActive
+     * 
+     * @param active new value
+     */
+    public void setBoardActive() {
+        this.boardIsActive = activateBoardForUser();
+    }
+
+    public boolean getUserIsRecruiter() {
+        return this.userIsRecruiter;
+    }
+
+    public boolean getLocalPlay() {
+        return this.localPlay;
+    }
+
     private void initController(Game gameState) {
         // Create turn order
         // This controller will use this to know which player controls what unit
         int agentIterator = 1; // This is in case there are less than four agents. Every unit will still be
+        System.out.println(localName);
```

Comment:
```suggestion
```

Reply:
know ive left some prints in mai aswell, but i figured whenever I find one I'll remove it :D:D:D